prun.py

```diff
@@ -30,9 +30,19 @@
 logging.basicConfig(filename='paracelrun_log', format = '%(asctime)s : %(levelname)s : %(message)s', level = logging.INFO)
 logger = logging.getLogger(__name__)
 
-PARACEL_INSTALL_PREFIX='./local/'
+PARACEL_INSTALL_PREFIX = os.path.dirname(os.path.abspath(__file__))
 #PARACEL_INSTALL_PREFIX='/nfs/xxx/paracel/local'
-#PARACEL_INSTALL_PREFIX='/usr/local'
+#PARACEL_INSTALL_PREFIX='./local'
```

Comment:
I don't this part is necessary. Once successfully installed, `$(PARACEL_INSTALL_PREFIX)/lib` should has been added to `/etc/ld.so.conf`